tests/unit/package_managers/yarn/test_main.py

```diff
@@ -16,20 +18,41 @@
 from cachi2.core.rooted_path import RootedPath
 
 
+class YarnVersions(Enum):
+    YARN_V1 = semver.VersionInfo(1, 0, 0)
+    YARN_V2 = semver.VersionInfo(2, 0, 0)
+
+    YARN_V3_RC1 = semver.VersionInfo(3, 0, 0, prerelease="rc1")
```

Comment:
Are we including this in any unit test?

Reply:
Yes,
 `test_main.py::test_yarn_unsupported_version_fail[package_manager,yarn_path-(3.0.0-rc1, None)]`

it'll get into `parametrize` as a set difference of supported and unsupported yarn versions (it was easier to have an enum just to have list comprehensions for parametrize)

Reply:
Right, I overlooked that. LGTM then!